### DISCUSSION TITLE
Scinterm 3.1 Released

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/scinterm#10] [General]
Scinterm 3.1 is released from https://github.com/orbitalquark/scinterm/releases/tag/scinterm_3.1

Scinterm no longer uses the LongTerm3 branch of Scintilla now that Scintilla 5 is considered stable from a platform perspective. 5.1.0 is the new minimum requirement.

The build process for Scinterm has slightly changed. `make patch` should be run before `make` in order to apply Scinterm-specific requirements to the Scintilla source. These patches are necessary until they are accepted upstream.

Bugfixes:

* None.

Changes:

* Added support for native terminal caret and added `scintilla_update_cursor()`.

**Comment by rhaberkorn** [comment]
Are there any plans to upstream Scinterm into Scintilla (so it becomes a permanently maintained port of Scintilla)?

**Comment by orbitalquark** [comment]
Neil does not want to maintain this platform, so unfortunately this project will remain separate for the forseeable future.
